<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyCompany</class>
 <widget class="QDialog" name="modifyCompany">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tabelle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_table"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_download">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Alle herunterladen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_companyName"/>
     </item>
     <item>
      <widget class="QComboBox" name="cb_rank"/>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_gold">
       <property name="text">
        <string>Gold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_silver">
       <property name="text">
        <string>Silber</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_bronze">
       <property name="text">
        <string>Bronze</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tv_company">
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>modifyCompany</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>modifyCompany</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
